packages/batch/lib/helper.js

```diff
@@ -18,12 +18,14 @@ class Helper {
     }
     const fuse = new Fuse(textToSearch, { ignoreLocation: true, includeScore: true})
     const matches = fuse.search(patternToMatch)
+
     if (matches.length === 0) {
       return false
     }
 
     const closestMatch = matches[0]
-    if (!closestMatch.score) {
+    // perfect match has a score of O, this is wrong !closestMatch.score
+    if (!closestMatch.hasOwnProperty('score')) {
       return false
     }
```

Comment:
This might not be required, cause in line 19, includeScore is true, which means it always will return the score property

Reply:
you are right